s27816-MichaelWajda/zajecia1/instalacja_debian.txt

```diff
@@ -0,0 +1,454 @@
+physical volume for LVM - umożliwiającym elastyczne zarządzanie partycjami. 
+LVM opisane w punkcie 25.
+do not use the partition - partycja/pamięć pozostanie niewykorzystana.
+Mount Options: Opcje montowania dysku
+discard - informuje system, aby przekazywał informacje o danych usuniętych do 
+dysku SSD za pomocą procesu TRIM. Pomaga to w utrzymaniu wydajności i 
+trwałości dysku poprzez oznaczanie bloków danych jako niepotrzebne, 
+co przyspiesza proces czyszczenia i przygotowania do ponownego zapisu.
+noatime - (no access time), wyłącza aktualizację ostatniej daty dostępu. 
+pomaga to zmniejszyć liczbę operacji zapisu na dysku, co z kolei może 
+poprawić wydajność systemu.
+nodiratime - (no directory access time), jak noatime, ale dotyczy tylko 
+katalogów.
+realtime - pozwala na utworzenie obszaru na dysku, który jest zarezerwowany 
+dla operacji związanych z przetwarzaniem danych w czasie rzeczywistym.
+nodev - zapobiega montowaniu urządzeń specjalnych wewnątrz danego systemu 
+plików (jak /dev/zero). Poprawia bezpieczeństwo przez ograniczenie dostępu.
+nosuid - zapobiega wykonywaniu plików z flagą SUID (Set User ID) lub SGID 
+(Set Group ID) wewnątrz danego systemu plików. Poprawia bezpieczeństwo.
+noexec - zapobiega uruchamianiu plików wykonywalnych z danego systemu plików, 
+co może zwiększyć bezpieczeństwo systemu poprzez ograniczenie możliwości 
+uruchamiania złośliwego oprogramowania.
+ro - (read only) Opcja montowania ustawiająca partycję tylko do odczytu.
+sync - zapewnia większe bezpieczeństwo danych, poprzez natychmiastowe 
+zapisywanie ich na dysku (kosztem wydajności).
+userquota - umożliwia kontrolę i monitorowanie zużycia przestrzeni dyskowej 
+na poziomie użytkownika dla danej partycji.
+grpquota - umożliwia kontrolę i monitorowanie zużycia przestrzeni dyskowej 
+na poziomie grupy użytkowników dla danej partycji.
+user_xattr - umożliwia korzystanie z rozszerzonych atrybutów użytkownika na 
+danej partycji, co pozwala na przechowywanie dodatkowych metadanych plików.
+Reserved blocks: ilość bloków na każdej partycji zarezerwowanych dla 
+root'a, zostawiamy bez zmian.
+Configure ISCSI volumes -  konfiguracja dysku, do którego nie mamy fizycznie, 
+ale do którego możemy uzyskać dostęp przez sieć (np. serwer firmy).
+29) Po dokonaniu partycjonowana wybieramy 'Finish partitioning and write 
+changes to disk' a następnie 'Yes' aby rozpocząć instalację podstawy 
+systemu (z nośnika). 
+Nie chcemy wybierać kolejnej płyty, więc zapytani wybieramy 'No', aby 
+rozpocząć ściąganie reszty systemu z sieci.
+Wybieramy terytorium na którym się znajdujemy 'Poland' a następnie 
+ftp.task.gda.pl, znajdującego się na politechnice Gdańskiej, czyli 
+najbliżej w tym przypadku.
+Proxy nie używamy, zostawiamy więc pusta przestrzeń i przechodzimy dalej.
+Na zapytanie o wzięcie udziału w ankiecie popularności pakietów wybieramy 
+'No', gdyż jest to instalacja w celu przeprowadzenia jedynie w celu 
+przeprowadzenia ćwiczenia.
```

Comment:
Skoro przeprowadzasz ćwiczenie i nie będziesz później korzystać z tej instalacji to po co było przenzaczać tyle przestrzeni dyskowej na coś co nie będzie używane? To marnotrawienie zasobów.